### ISSUE TITLE
Internal links RSS feed remain <saber-link>

### ISSUE THREAD

**Issue by spekulatius** [saberland/saber#246]
- [x] I confirm that this is an issue rather than a question.

## Bug report

It appears as internal links in feeds aren't compiled to a-tags.

See https://saber-blog-starter.netlify.com/atom.xml

#### Version

Saber 0.6.7

info Direct dependencies
├─ prismjs@1.16.0
├─ saber-highlight-css@0.0.3
├─ saber-plugin-feed@0.3.2
├─ saber-plugin-google-analytics@0.0.6
├─ saber-plugin-prismjs@0.2.1
├─ saber-plugin-pwa@0.3.3
├─ saber-plugin-query-posts@0.3.2
├─ saber-plugin-sitemap@0.2.0
├─ saber-theme-minima@3.0.1
└─ saber@0.6.7

#### Steps to reproduce

Code can be found on https://github.com/spekulatius/saber-blog-starter - current master branch.

#### What is expected?

A valid HTML a-tag, so that people can click on internal links and still get to the intended target.

#### What is actually happening?

Links like this 
```
<saber-link :to="$saber.getPageLink(&quot;./index.md&quot;, &quot;&quot;)">Back Home</saber-link>
``` 

remain in the output

#### Other relevant information

- Result of running `saber -v`: saber/0.6.7 linux-x64 node-v8.10.0
- Browser version (optional): FF 67.0
- Is Saber a global or local install? locally
- Which package manager did you use for the install? yarn
- Does this issue occur when all plugins are disabled? -

**Comment by egoist** [comment]
Waiting for https://github.com/saberland/saber/issues/204 to be implemented

**Comment by spekulatius** [comment]
Ah, okay. I wasn't aware of the open issue.

**Comment by h404bi** [comment]
The issue is fixed by #268 and could be closed.
